src/main/java/dk/dbc/lobby/rest/ApplicantsResource.java

```diff
@@ -264,6 +264,18 @@ public Response getApplicantBody(@PathParam("id") String id) {
         return Response.ok().type(applicantEntity.getMimetype()).entity(streamingOutput).build();
     }
 
+    @GET
+    @Path("/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getApplicant(@PathParam("id") String id) {
+        final ApplicantEntity applicantEntity = entityManager.find(ApplicantEntity.class, id);
+        if (applicantEntity == null) {
+            return Response.status(410).entity("Applicant not found").build();
```

Comment:
Du kan jo strengt taget ikke vide om den er "gone" eller "not found"  (men no worries :-) )